### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by MaksymKos**

### CODE REVIEW

File: src/calculateRentalCost.js [hunk 1]
```diff
@@ -28,7 +28,23 @@
...
+    rent = days * 40;
+
+    return rent;
+  }
+
+  if (days < 7) {
+    rent = (days * 40) - 20;
+
+    return rent;
+  } else {
```
Comment (tetiana-ivanchenko) [inline src/calculateRentalCost.js:43]: else is redundant

File: src/calculateRentalCost.js [hunk 2]
```diff
@@ -28,7 +28,23 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let rent = 0;
+
+  if (days <= 2) {
+    rent = days * 40;
+
+    return rent;
```
Comment (tetiana-ivanchenko) [inline src/calculateRentalCost.js:36]:
```suggestion
    return days * 40;
```

Comment (tetiana-ivanchenko) [reply src/calculateRentalCost.js:36]: fix it everywhere

File: src/calculateRentalCost.js [hunk 3]
```diff
@@ -28,7 +28,23 @@
  */
 
 function calculateRentalCost(days) {
```
Comment (tetiana-ivanchenko) [inline src/calculateRentalCost.js:30]: move all magic numbers in the variables

**Review by tetiana-ivanchenko — CHANGES_REQUESTED** [review]

**Review by Viktor-Kost — APPROVED** [review]
Nice!
And i left one comment to improve your code

**Pull request closed without merging**